include/revng/Model/Binary.h

```diff
@@ -118,6 +118,16 @@ class model::Binary : public model::generated::Binary {
 
   model::TypePath recordNewType(UpcastablePointer<Type> &&T);
 
+  /// \todo limit \tparam NewType using `SpecializationOf<model::Type>` concept
+  /// when it's available (after updating to clang-13+).
+  template<typename NewType>
+  std::pair<NewType &, model::TypePath> newType() {
```

Comment:
* Nice!
* `makeType()`

Reply:
I forgot to close this one too 😅.
https://github.com/revng/revng/pull/232/commits/61c53501df72b0d13ce0ba0ef11541c973a09fa7